.env-dist

```
@@ -120,4 +123,5 @@ E2E_TEST_ACCOUNT_PASSWORD=
 # Monitor Premium features
 PREMIUM_ENABLED=
 # Link to start user on the subscription process. PREMIUM_ENABLED must be set to `true`.
-SUBSCRIBE_PREMIUM_URL=
+PREMIUM_PRODUCT_ID=prod_NErZh679W62lai
+PREMIUM_PLAN_ID_US=price_1MUNq0Kb9q6OnNsL4BoJgepf
```

@rhelmer I just saw somewhere that we might even have monthly and annual plans, so we'd probably want to signify that too:

```suggestion
PREMIUM_PLAN_ID_US_MONTHLY=price_1MUNq0Kb9q6OnNsL4BoJgepf
```